src/constants.d.ts

```diff
@@ -360,6 +360,9 @@ export const NotEqualStencilFunc: StencilFunc;
 export const GreaterEqualStencilFunc: StencilFunc;
 export const AlwaysStencilFunc: StencilFunc;
 
+/**
+ *  @see {@link https://developer.mozilla.org/en-US/docs/Web/API/WebGL_API/Constants#Buffers|WebGL API Constant}
+ */
```

Comment:
Not sure we should start to link to MDN for all possible WebGL constants. You could basically do this for all enums in `constants.d.ts` and I have the feeling that's a bit over-engineering.

Reply:
> link to MDN for all possible WebGL constants

It is a little hard, and take a long time to do it. I don't know how write down it for document reader, may be it don't be writed down now.